# HG changeset patch
# User adf88@interia.pl
# Date 1501222009 -7200
#      Fri Jul 28 08:06:49 2017 +0200
# Branch trunk
# Node ID 6a88921a3f3b44f25d0bac9c38ecb34cf61c87c4
# Parent  096954668ebcc43ef2c7ccdfb7c7d42a472df1cc
Fix: Update vehicle orders window when last vehicle is removed from shared orders group.

diff -r 096954668ebc -r 6a88921a3f3b src/vehicle.cpp
--- a/src/vehicle.cpp	Thu Jul 13 17:45:38 2017 +0000
+++ b/src/vehicle.cpp	Fri Jul 28 08:06:49 2017 +0200
@@ -2677,7 +2677,7 @@
 	if (this->orders.list->GetNumVehicles() == 1) {
 		/* When there is only one vehicle, remove the shared order list window. */
 		DeleteWindowById(GetWindowClassForVehicleType(this->type), vli.Pack());
-		InvalidateVehicleOrder(this->FirstShared(), 0);
+		InvalidateVehicleOrder(this->FirstShared(), VIWD_MODIFY_ORDERS);
 	} else if (were_first) {
 		/* If we were the first one, update to the new first one.
 		 * Note: FirstShared() is already the new first */
